_config/cms.yml

```diff
@@ -5,8 +5,7 @@ Only:
 ---
 SilverStripe\Core\Injector\Injector:
   SilverStripe\CMS\Controllers\CMSPageHistoryController:
-    factory:
-      SilverStripe\VersionedAdmin\Controllers\HistoryControllerFactory
+    class: SilverStripe\VersionedAdmin\Controllers\CMSPageHistoryViewerController
```

Comment:
This will break any implementations that use the existing API to opt out of elemental using this new feature (plus any third party modules plus user code that does the same). I think it’s pretty low risk and worth doing anyway but figure it’s worth noting.

Reply:
Yeah ... that was my thought as well.

If you care enough about what history viewer you are using to customise it, you'll probably want to update your customisation based on the updated availability `CMSPageHistoryViewerController`.